src/main/kotlin/domain/RacingGame.kt

```diff
@@ -0,0 +1,17 @@
+package domain
+
+import util.RandomNumber
+
+class RacingGame(
+    private var cars: List<Car>
+) {
+
+    fun getCars() = cars
+
+    fun racing(): List<Car> {
+        cars.map { car ->
```

Comment:
```suggestion
        cars.foreach { car ->
```

foreach가 더 좋지 않을까요~?